backend/user-service/.env.sample

```diff
@@ -0,0 +1,2 @@
+MONGO_URI="maxhasntsendyet"
+PORT=3000
```

Comment:
use port 4000 instead since zac's question service implementation uses 3000

Reply:
also the default port if PORT is not specified is 4000 as in config.ts

Reply:
Ok fixed.